charts/hawkbit/values.yaml

```diff
@@ -83,6 +83,7 @@ fileStorage:
 env:
   springDatasourceHost: "hawkbit-mysql"
   springDatasourceDb: "hawkbit"
+  springDatasourceUrl: ""
```

Comment:
I think a comment would be helpful here, pointing out that if `springDatasourceUrl` is set, it takes precedence over the URL created from default mysql DB host (if `mysql.enabled`) or `springDatasourceHost` and `springDatasourceDb`.